cmake/Buildwearables.cmake

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2018  iCub Facility, Istituto Italiano di Tecnologia
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+find_or_build_package(YARP QUIET)
+
+set(WEARABLES_DEPENDS "")
+list(APPEND WEARABLES_DEPENDS YARP)
+
+if(WIN32)
+    find_or_build_package(ICUB QUIET)
+    find_or_build_package(icub-firmware-shared QUIET)
+    find_or_build_package(forcetorque-yarp-devices QUIET)
+
+    list(APPEND WEARABLES_DEPENDS ICUB)
+    list(APPEND WEARABLES_DEPENDS icub-firmware-shared)
+    list(APPEND WEARABLES_DEPENDS forcetorque-yarp-devices)
+endif()
```

Comment:
Can you explain a bit more about this? `wearables` should compile fine also without all this software, right? If there is just a runtime dependency, I think we can avoid adding this software as a dependency to wearables, because a superbuild dependency is typically intended as **compile time** dependency, i.e. that wearables will not compile without this project installed.

Reply:
Correct me if I am wrong. You are saying that, we have these repos `ICUB`, `icub-firmware-shared`, and `forcetorque-yarp-devices` already in `superbuild` when we enable the `ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS `. And we need them at runtime to run the `wearables` smoothly without problem. But at compile time, `wearables` does not depend on them; therefore, it can be compiled even without those repos? Therefore, I should remove them as explicit dependencies of `wearables`?

Reply:
Exactly. So if someone wants to use `wearables` without installing icub and the other repo he can without problems.